doc/articles/uwp-vs-winui3.md

```diff
@@ -39,7 +39,7 @@ The questions below should help guide you to the best choice when creating a new
 
 ### Do you use 3rd-party libraries?
 
-While 3rd party vendors have offerings for both UWP and WinUI, UWP has been in the market longer. From Uno Platform perspective either will work, or if not, contact us at our [Discord channel](https://discord.com/invite/eBHZSKG).
+While 3rd party vendors have offerings for both UWP and WinUI, UWP has been in the market longer. From Uno Platform perspective either will work, or if not, contact us at our [Discord channel](https://platform.uno/discord).
```

Comment:
Thanks, @agneszitte. I somehow missed this 🤔 Just for a bit of standardization, I suggest  
```suggestion
While 3rd party vendors have offerings for both UWP and WinUI, UWP has been in the market longer. From Uno Platform perspective either will work, or if not, contact us on our [Discord Server](https://platform.uno/discord).
```

Reply:
thanks @morning4coffe-dev !